src/osp/Active/SysPrefabInit.cpp

```diff
@@ -121,19 +131,25 @@ void SysPrefabInit::init_drawing(
 
         for (std::size_t i = 0; i < objects.size(); ++i)
         {
+            ActiveEnt const ent = (*itPfEnts)[i];
+
+            // TODO: Don't actually do this. This marks every entity as drawable,
```

Comment:
??

Reply:
????

what part of this doesn't make sense?